contracts/modules/uniswap/v2/TernaryLib.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.5.0;
+
+/// @title Library for replacing ternary operator with efficient bitwise operations
+library TernaryLib {
+    /// @notice Equivalent to the ternary operator: `condition ? a : b`
+    function ternary(bool condition, uint256 a, uint256 b) internal pure returns (uint256 res) {
```

Comment:
not used anywhere either?

Reply:
It was more of an illustration. Now that overloaded versions of `ternary()` on `int256` and `address` are used in `V3SwapRouter`, should we leave this one here for completeness?